app/code/Magento/Cms/Model/ResourceModel/Page.php

```diff
@@ -126,6 +126,10 @@ protected function _beforeSave(AbstractModel $object)
                 __('The page URL key cannot be made of only numbers.')
             );
         }
+
+        $date = (new \DateTime())->format(DateTime::DATETIME_PHP_FORMAT);
```

Comment:
Please, use framework implementation for date value \Magento\Framework\Stdlib\DateTime\DateTime::date